_scripts/update-external-docs.rb

```diff
@@ -40,6 +40,10 @@
   'https://github.com/cockpit-project/cockpit/blob/master/HACKING.md' =>
   '{{ site.baseurl }}/external/source/HACKING.html',
 
+  './pkg/' => 'https://github.com/cockpit-project/cockpit/tree/main/pkg',
```

Comment:
Is the left side an exact match or substring? You are replacing `pkg/` with `pkg`, so a `pkg/foo` would be substituted with `pkgfoo`. Probably safer to consistently append a slash or not.